src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -222,9 +235,20 @@ export default {
 			if (this.isLastMessage) {
 				return false
 			}
-			return (!this.isCollapsedSystemMessage && this.message.id === this.visualLastLastReadMessageId)
-				|| (this.isCollapsedSystemMessage && this.message.id === this.visualLastLastReadMessageId && this.message.id !== this.lastCollapsedMessageId)
-				|| (this.isCombinedSystemMessage && this.lastCollapsedMessageId === this.visualLastLastReadMessageId)
+
+			if (this.message.id === this.visualLastLastReadMessageId) {
+				return !this.isCollapsedSystemMessage || this.message.id !== this.lastCollapsedMessageId
+			}
+
+			return this.isCombinedSystemMessage && this.lastCollapsedMessageId === this.visualLastLastReadMessageId
+		},
+
+		shouldShowSummaryOption() {
+			if (!canSummarizeChat || !this.isLastReadMessage
```

Comment:
```suggestion
			if (!canSummarizeChat
```

It's already in the unread marker div